src/main/java/gregtech/common/pipelike/cable/tile/CableEnergyContainer.java

```diff
@@ -85,7 +85,7 @@ private long dispatchEnergyToNode(BlockPos nodePos, int nodeBlockedConnections,
             if (tileEntity == null || tileEntityCable.getPipeBlock().getPipeTileEntity(tileEntity) != null) {
                 continue; //do not emit into other cable tile entities
             }
-            IEnergyContainer energyContainer = tileEntity.getCapability(GregtechCapabilities.CAPABILITY_ENERGY_CONTAINER, null);
+            IEnergyContainer energyContainer = tileEntity.getCapability(GregtechCapabilities.CAPABILITY_ENERGY_CONTAINER, facing.getOpposite());
```

Comment:
LAGIdiot, can you explain to me what this does?  (What was wrong with null?)

Reply:
Sure, when getCapability does not know (null supplied) which side of tile entity it should check it will just ignore all covers capabilities on it.

Reply:
Ultimately what was the result to the end user?

Reply:
Result was energy network did not check that it was connected to machine on Shutter Cover which should prevent all connection. And it blowed machine up as MV cable was connected to LV machine.